.github/workflows/demo_in_readme.yaml

```diff
@@ -85,9 +101,12 @@ jobs:
     needs: check-requirements
     runs-on: [lmtest]
     steps:
+    - name: mask env 
+      run: |
+        echo "::add-mask::${{env.WORKSPACE_PREFIX}}"
     - uses: actions/checkout@v3
 
     - name: chat-model-in-hf
       run: |
         source activate internlm-env-test
-        srun -p llm python ./ci_scripts/model/demo_load_7B_chat_model.py
+        nohup srun -p llm python ./ci_scripts/model/demo_load_7B_chat_model.py 2>&1 | tee -a ${GITHUB_JOB}.log
```

Comment:
set job name for each job when submiting ci tasks, and use squeue to check the job status with job name, instead of parsing the running log.